.gitignore

```diff
@@ -44,7 +44,6 @@ bin/
 lerna-debug.log
 cactus-openapi-spec.json
 cactus-openapi-spec-*.json
-.npmrc
```

Comment:
@outSH The .npmrc file can contain tokens and therefore is strongly recommended to not keep it under version control. 
Even if there are no tokens in somebody's own .npmrc file, the fact that they have one will give them all sorts of grief if we now start committing one and leave them to manually deal with that every single time they make a contribution.

My recommendation here is to:
1. Add a new npm script in the root package.json called `"init-registries"` that will do the same thing that the config file does now (`npm config set @iroha2:registry=https://nexus.iroha.tech/repository/npm-group/`)
2. Change the scripts (probably the configure script the right place for it but let me know if you have any other place you think it would be best) so that they invoke this other utility script from 1) to set the npm config up with the registry informaiton.
3. Delete the .npmrc file you committed
4. Revert the change that makes git not ignore the .npmrc files

This way I'm hoping that everything will still work transparently to contributors, as-in, there will be no extra manual configuration step involved when setting up the project build, but without the problem of us committing the .npmrc file and then condemning everybody to having to deal with that.

Reply:
@petermetz Yeah, I agree, I was kinda hoping they'll release these packages sooner and was aiming for simplest solution now, didn't think about secret leaking though. 

I've followed your suggestion (and additionally added `init-registries`  to `reset:yarn-lock` since it uses yarn install). The two problems I see:
- `npm config` adds entry into user `~/.npmrc`, not project one. I think silently modifying user environment is not a very good practice, but the harm and possible drawback is small so I think it's OK (we also have script for installing yarn globally, but this requires explicit user action).
- Calling `yarn install` directly, without running configure / reset:yarn-lock first will fail with fairly cryptic error. This is very rare I think, but we can keep and eye if someone doesn't report it.

If these are acceptable for you then we can merge I think :) The alternative would be to put .npmrc in iroha2 package only - that way the risk of leaking secrets is minimized and the process is transparent, but the current state seem OK as well (ideally, we'll remove it once Iroha V2 releases it's official packages so no use in dwelling over this for too long).

Reply:
@outSH Thank you very much for the updates! Agreed on both of these also being issues that you mentioned so I'm keeping my fingers crossed for a quick iroha v2 client library release!